### ISSUE TITLE
🐞 - Spam NFT cannot be burned

### ISSUE THREAD

**Issue by michikaa1** [bloomwalletio/bloom#2257]
### Version

1.0.0

### Description

Thu, 04 Apr 2024 05:42:26 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 05:42:20 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 05:42:08 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 05:41:24 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 01 Apr 2024 13:07:50 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 01 Apr 2024 13:07:36 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 01 Apr 2024 13:07:31 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 01 Apr 2024 13:07:22 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 01 Apr 2024 13:07:04 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Sun, 24 Mar 2024 17:47:55 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 21 Mar 2024 17:32:25 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 21 Mar 2024 17:30:17 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 21 Mar 2024 17:29:01 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Tue, 19 Mar 2024 19:56:54 GMT
error: `node error: The requested data https://shimmer-node.tanglebay.com/api/participation/v1/events/0xaa76a719998a4c37c7b70ce40d75f7256789264f9962c327798c25c6eb1c48c7 was not found.`

Wed, 13 Mar 2024 17:48:20 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:48:01 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:47:51 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:47:42 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:47:34 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:47:12 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:36:46 GMT
error: `node error: The requested data https://api.shimmer.network/api/participation/v1/events/0x69194943b788eb67bfd3b1e258041af1cbb8f501489c5d826e6cd970cfaf01eb was not found.`

Mon, 11 Mar 2024 18:57:43 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:55:40 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:54:03 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:49:34 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:48:37 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:48:15 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:47:55 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 08:10:39 GMT
error: `node error: The requested data https://shimmer-node.tanglebay.com/api/participation/v1/events/0xaa76a719998a4c37c7b70ce40d75f7256789264f9962c327798c25c6eb1c48c7 was not found.`

Mon, 11 Mar 2024 08:03:04 GMT
error: `node error: The requested data https://shimmer-node.tanglebay.com/api/participation/v1/events/0xaa76a719998a4c37c7b70ce40d75f7256789264f9962c327798c25c6eb1c48c7 was not found.`

### Expected behaviour

Spam NFT cannot be burned, etc.

### Actual behaviour

Spam NFT cannot be burned, etc.

### Can the issue reliably be reproduced?

Yes

### Crash Status

None of the above

### Error Messages

unfulfillable requirement Nft(Nftid(0x.....c5)

### Error Logs

```shell
Thu, 04 Apr 2024 06:08:08 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 06:07:51 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 06:06:24 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 06:05:47 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 06:05:20 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 06:04:58 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 05:42:26 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 05:42:20 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 05:42:08 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 04 Apr 2024 05:41:24 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 01 Apr 2024 13:07:50 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 01 Apr 2024 13:07:36 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 01 Apr 2024 13:07:31 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 01 Apr 2024 13:07:22 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 01 Apr 2024 13:07:04 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Sun, 24 Mar 2024 17:47:55 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 21 Mar 2024 17:32:25 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 21 Mar 2024 17:30:17 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Thu, 21 Mar 2024 17:29:01 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Tue, 19 Mar 2024 19:56:54 GMT
error: `node error: The requested data https://shimmer-node.tanglebay.com/api/participation/v1/events/0xaa76a719998a4c37c7b70ce40d75f7256789264f9962c327798c25c6eb1c48c7 was not found.`

Wed, 13 Mar 2024 17:48:20 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:48:01 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:47:51 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:47:42 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:47:34 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:47:12 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Wed, 13 Mar 2024 17:36:46 GMT
error: `node error: The requested data https://api.shimmer.network/api/participation/v1/events/0x69194943b788eb67bfd3b1e258041af1cbb8f501489c5d826e6cd970cfaf01eb was not found.`

Mon, 11 Mar 2024 18:57:43 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:55:40 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:54:03 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:49:34 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:48:37 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:48:15 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 18:47:55 GMT
wallet: `unfulfillable requirement Nft(NftId(0xb00788e0aaac0f313e738a00ae3abec40e499fef801b6ee5c1f7feb0df3f93c5))`

Mon, 11 Mar 2024 08:10:39 GMT
error: `node error: The requested data https://shimmer-node.tanglebay.com/api/participation/v1/events/0xaa76a719998a4c37c7b70ce40d75f7256789264f9962c327798c25c6eb1c48c7 was not found.`

Mon, 11 Mar 2024 08:03:04 GMT
error: `node error: The requested data https://shimmer-node.tanglebay.com/api/participation/v1/events/0xaa76a719998a4c37c7b70ce40d75f7256789264f9962c327798c25c6eb1c48c7 was not found.`
```

### Profile Type

Software (Stronghold)

### Operating System

Windows

### Diagnostics

```shell
Version: 1.0.0
Sprache: de
Währung: eur
Node-Liste: https://api.stardust-mainnet.iotaledger.net,https://iota-node.tanglebay.com
Plattform: win32
Plattformversion: 10.0.22631
Plattform-Architektur: x64
CPU-Anzahl: 8
Gesamtspeicher: 6058.2 MB
FrVersion: 1.0.0
Sprache: de
Währung: eur
Node-Liste: https://api.stardust-mainnet.iotaledger.net,https://iota-node.tanglebay.com
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Tuditi** [comment]
Hey @michikaa1 , was this issue resolved in the 1.0.2 release?

**Comment by michikaa1** [comment]
I created a new profile and the Spam nft had disappeared and is not longer present after 1.0.2 update Schöne GrüßeMichael ***@***.*** 25.04.24, 16:20 schrieb Tuditi ***@***.***>:

   
  Hey @michikaa1 , was this issue resolved in the 1.0.2 release?

**Comment by Tuditi** [comment]
Thank you! I will close this issue then.
